optuna/pruners/hyperband.py

```diff
@@ -49,18 +47,39 @@ class HyperbandPruner(BasePruner):
         Thus, for example, if ``HyperbandPruner`` has :math:`4` pruners in it,
         at least :math:`4 \\times 10` pruners are consumed for startup.
 
+    .. note::
+        Hyperband has several :class:`~optuna.pruners.SuccessiveHalvingPruner`. Each
+        :class:`~optuna.pruners.SuccessiveHalvingPruner` is referred as "bracket" in the original
+        paper. The number of brackets is an important factor to control the early stopping behavior
+        of Hyperband and is determined by ``max_resource`` and ``reduction_factor`` as
+        `n_brackets = floor(log(max_resource) / log(reduction_factor))`. Please set
```

Comment:
I believe we should "+1".

Reply:
Yes, you are right. I will fix it.

Reply:
Not a strong opinion but how about using a proper `:math:` formula?

Reply:
It's not rendered correctly it seems. Could you verify your changes either locally or on CircleCI's dashboard? https://circleci.com/gh/optuna/optuna/37699#artifacts/containers/0